tree_test.go

```diff
@@ -1075,7 +1075,7 @@ var (
 		{197, 210, 70, 1, 134, 247, 35, 60, 146, 126, 125, 178, 220, 199, 3, 192, 229, 0, 182, 83, 202, 130, 39, 59, 123, 250, 216, 4, 93, 133, 164, 112},
 		{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0},
 	}
-	testAccountRootCommRust, _ = hex.DecodeString("10ed89d89047bb168baa4e69b8607e260049e928ddbcb2fdd23ea0f4182b1f8a")
+	testAccountRootCommRust, _ = hex.DecodeString("4caf7631af042c1615845a0116e789de0beea3e07f4b947377101aa822964614")
```

Comment:
This has been changed in this PR: https://github.com/crate-crypto/rust-verkle/pull/85